packages/synapse-react-client/src/components/dataaccess/AccessRequirementTable.integration.test.tsx

```diff
@@ -138,9 +86,16 @@ describe('Access Requirement Table tests', () => {
 
     renderComponent(props)
 
-    await waitFor(() => expect(onServiceRecievedRequest).toHaveBeenCalled())
```

Comment:
Verifying that the service was called can be brittle, so remove that check and replace with a check to verify that the table is shown with the expected number of rows.